src/vue/components/popup.vue

```diff
@@ -233,7 +233,7 @@ export default {
         return '0px'
       }
 
-      return this.moduleConfig.initialPos.left
+      return `${this.moduleConfig.initialPos.left}px`
```

Comment:
do we have a test somewhere that ensures that the settings in moduleConfig are just numeric?

Reply:
I am not sure - it comes as property, so we could add this check inside it loads.
I am not sure, we should place it here.